servers/zms/src/main/java/com/yahoo/athenz/zms/store/jdbc/JDBCConnection.java

```diff
@@ -647,7 +650,9 @@ public boolean deleteDomain(String domainName) {
 
     PreparedStatement prepareDomainScanStatement(String prefix, long modifiedSince)
             throws SQLException {
-        
+
+        Calendar cal = Calendar.getInstance(TimeZone.getTimeZone(MYSQL_SERVER_TIMEZONE));
```

Comment:
let's move this code how it was before. as I mentioned in my earlier comment we don't need a calendar object if we're just doing a list domain operation.